### ISSUE TITLE
Error when adding templates

### ISSUE THREAD

**Issue by kbrandwijk** [graphcool/graphcool#813]
![image](https://user-images.githubusercontent.com/852069/31577393-b3eda2aa-b10d-11e7-8384-72cec9487b89.png)

Yarn is available, because it works if I run it manually:
![image](https://user-images.githubusercontent.com/852069/31577397-c9372afa-b10d-11e7-80c1-83e0f4cec74a.png)

graphcool/0.7.0 (windows-x64) node-v8.4.0

**Comment by delgermurun** [comment]
I got this error on mac (High Sierra)

```
Downloading template graphcool/templates/authentication/email-password from graphcool/templates ... ✔
  module Downloaded graphcool/templates to /var/folders/md/xj_ds9bx5xj4fc9bytltpgd40000gn/T/cj8s157c90000mvanrszolbew/ +38s
Path .../chat/server/src/email-password/ already exists. Overwriting it now.
Error: ENOENT: no such file or directory, stat '/var/folders/md/xj_ds9bx5xj4fc9bytltpgd40000gn/T/cj8s157c90000mvanrszolbew/authentication/email-password/src'
    at Object.fs.statSync (fs.js:955:11)
    at Object.statSync (.../.nvm/versions/node/v8.7.0/lib/node_modules/graphcool/node_modules/graceful-fs/polyfills.js:297:22)
    at Object.copySync (.../.nvm/versions/node/v8.7.0/lib/node_modules/graphcool/node_modules/fs-extra/lib/copy-sync/copy-sync.js:31:86)
    at AddTemplate.<anonymous> (.../.nvm/versions/node/v8.7.0/lib/node_modules/graphcool/node_modules/graphcool-cli-core/src/commands/add-template/add-template.ts:81:8)
    at step (.../.nvm/versions/node/v8.7.0/lib/node_modules/graphcool/node_modules/graphcool-cli-core/dist/commands/add-template/add-template.js:42:23)
    at Object.next (.../.nvm/versions/node/v8.7.0/lib/node_modules/graphcool/node_modules/graphcool-cli-core/dist/commands/add-template/add-template.js:23:53)
    at fulfilled (.../.nvm/versions/node/v8.7.0/lib/node_modules/graphcool/node_modules/graphcool-cli-core/dist/commands/add-template/add-template.js:14:58)
    at <anonymous>
Exiting with code: 1
```

**Comment by kbrandwijk** [comment]
The correct path is: `gc add-template graphcool/templates/auth/email-password`. The error should be clearer though.

**Comment by timsuchanek** [comment]
Thanks for reporting @kbrandwijk !
I improved the bin path resolution for the `yarn` and `npm` binary.
This is now published at `graphcool@0.8.0-alpha.7`.
Can you confirm that it's now working?

**Comment by timsuchanek** [comment]
@delgermurun For better template error handling, stay tuned in #819.
